### PR TITLE
feat(bookmarks): adds rofi bookmarks selector

### PR DESCRIPTION
**Pull request by rubiin**
image

Adding this script as part of hyde cli.
Only supports chrome based browsers like brave, vivaldi, chromium and google-chrome

### CODE REVIEW

**Comment by rubiin** [comment]
Also might want to theme it after merge to look like cliphist

File: Scripts/Launch-Bookmarks [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+fi
+
+if [[ $BROWSER == "google-chrome-stable" ]]; then
+	BOOKMARKS_PATH="$HOME/.config/google/Default/Bookmarks"
+fi
+
+selected=$(grep '"url":' "$BOOKMARKS_PATH" | awk '{print $2}' | sed 's/"//g' | rofi -dmenu -p "Select a Browser Bookmark")
+
+if [ "$selected" ]; then
+	$BROWSER "$selected"
```
Comment (kRHYME7) [inline Scripts/Launch-Bookmarks:24]: use ` xdg-open ` instead of  the browser

Comment (rubiin) [reply Scripts/Launch-Bookmarks:24]: oh we can do that. I am testing out some rofi styling

**Comment by rubiin** [comment]
I just threw in bunch of stuffs from another glyph luancher

**Review by kRHYME7 — CHANGES_REQUESTED** [review]

**Comment by kRHYME7** [comment]
Will do a rewrite to handle other browsers

**Comment by kRHYME7** [comment]
image

I revised it into using the default set browser and recursively search for */Default/Bookmarks path(handles flatpak).

**Comment by rubiin** [comment]
yeah for chromium you can do that. By that it would handle almost all chrome browsers. For mozilla firefox https://github.com/milosz/rofi-firefox-bookmarks/blob/main/rofi_firefox_bookmarks.sh see this

**Comment by kRHYME7** [comment]
Can you checkout the latest commit?

**Comment by rubiin** [comment]
beat you to it 
image

works as intended

**Comment by kRHYME7** [comment]
I will remove the ` lz4json ` dep  following the link you send. Thanks! Did not know about this.

**Comment by rubiin** [comment]
Regarding tui, lets do on golang . Using this https://charm.sh

**Pull request merged**